dist/src/main/config/application.yaml

```diff
@@ -87,3 +87,30 @@ zeebe:
       # Controls the number of io threads to be used.
       # This setting can also be overridden using the environment variable ZEEBE_BROKER_THREADS_IOTHREADCOUNT
       ioThreadCount: 2
+
+# Operate configuration properties
+
+camunda.operate:
```

Comment:
🤔 Interesting so Operate starts actually with a `camund` prefix.

Reply:
Yes, to my knowledge, all webapps start with `camunda`.

Reply:
Do we need to change this at some point for Zeebe?

Reply:
Yes, we will harmonize the configurations between the components. It should be tackled with a separate issue.